pkg/api/validation/validation.go

```diff
@@ -67,26 +68,6 @@ var PortNameErrorMsg string = fmt.Sprintf(`must be an IANA_SVC_NAME (at most 15
 
 const totalAnnotationSizeLimitB int = 256 * (1 << 10) // 256 kB
 
-func ValidateLabelName(labelName string, fldPath *field.Path) field.ErrorList {
```

Comment:
You may add a .import-restriction file (search the repo for examples) if you wish to document/enforce this. :)